x-pack/plugins/infra/public/pages/metrics/hosts/components/kpi_charts/tile.tsx

```diff
@@ -17,13 +17,16 @@ interface Props extends Omit<ChartBaseProps, 'type'> {
 export const Tile = ({ type, ...props }: Props) => {
   const { baseRequest } = useHostsViewContext();
 
-  const { nodes, loading } = useSnapshot({
-    ...baseRequest,
-    metrics: [{ type }],
-    groupBy: null,
-    includeTimeseries: true,
-    dropPartialBuckets: false,
-  });
+  const { nodes, loading } = useSnapshot(
+    {
+      ...baseRequest,
+      metrics: [{ type }],
+      groupBy: null,
+      includeTimeseries: true,
+      dropPartialBuckets: false,
+    },
+    { abortable: true }
```

Comment:
**thought**: shouldn't we make the default behaviour to abort the request, as this is an expensive one?

Reply:
Snapshot API wasn't really designed to work in the Hosts view, so it's very expensive in this context, but not so much in Inventory UI, and I also wanted to limit the scope of this PR. We can get back to this in a follow-up PR, wdyt?